src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const body = document.querySelector('.page__body');
+const menuOpener = document.querySelector('.icon--menu');
+const menuCloser = document.querySelector('.icon--cross');
+
+menuOpener.addEventListener('click', function() {
+  body.classList.add('page__body--with-menu');
+});
+
+menuCloser.addEventListener('click', () => {
+  body.classList.remove('page__body--with-menu');
+});
```

Comment:
please add this script (updated)

https://mateacademy.slack.com/archives/C029U4B5CUE/p1631011249064300
https://github.com/mate-academy/layout_miami